core/designsystem/src/main/java/com/google/samples/apps/nowinandroid/core/designsystem/component/LoadingWheel.kt

```diff
@@ -142,5 +161,22 @@ fun NiaLoadingWheelPreview() {
     }
 }
 
+@Preview(
```

Comment:
Small nit: We have a `@ThemePreviews` private to `core/designsystem`, would be great to reuse that here.